src/main/java/org/terasology/inGameHelp/InGameHelpCategoryRegistry.java

```diff
@@ -17,7 +17,20 @@
 
 import org.terasology.inGameHelp.systems.HelpCategory;
 
+/**
+ * Interface for managing the different help categories
+ */
 public interface InGameHelpCategoryRegistry {
+
+	/**
+	 * @return a list of the help categories
```

Comment:
for the sake of completeness, please add one sentence in addition to `@return`, like 
```
"Retrieve the {@link org.terasology.inGameHelp.systems.HelpCategory}s previously registered to this registry."
```